great_expectations/expectations/metrics/util.py

```diff
@@ -570,7 +570,9 @@ def column_reflection_fallback(
         if isinstance(selectable, TextClause):
             query: TextClause = selectable
         else:
-            query: Select = sa.select([sa.text("*")]).select_from(selectable).limit(1)
+            query: Select = (
+                sa.select([sa.text("*")]).select_from(sa.text(selectable)).limit(1)
+            )
```

Comment:
Hi @tommy-watts-depop - thanks again for this contribution. I think we will want to limit the effect to just the Redshift dialect, so would you mind making this change? After that, tests should pass and we will be good to merge this PR.

```suggestion
            if dialect.name.lower() == GESqlDialect.REDSHIFT:
                # Redshift needs temp tables to be declared as text
                query: Select = (
                    sa.select([sa.text("*")]).select_from(sa.text(selectable)).limit(1)
                )
            else:
                query: Select = (
                    sa.select([sa.text("*")]).select_from(selectable).limit(1)
                )
```

Reply:
You'll also need to add this import: `from great_expectations.execution_engine.sqlalchemy_dialect import GESqlDialect`

Reply:
Done

Reply:
Wow - rapid response! I think you also need to commit the suggestion in addition to adding the import.

Reply:
@anthonyburdi Apologies forgot to save the file ! repushed